data/projects/a/arbilearn.yaml

```diff
@@ -0,0 +1,16 @@
+version: 7
+name: ArbiLearn
+display_name: ArbiLearn
+description: ArbiLearn is focused on advancing the Arbitrum ecosystem through educational resources. It aims to develop courses and interactive tutorials on Arbitrum’s technology, including its Layer 2 solutions. By enhancing developer knowledge and providing practical guidance, ArbiLearn supports the broader adoption and integration of Arbitrum technology.
+websites:
+  - https://github.com/aliveevie/ArbiLearn
+social:
+  - twitter: https://twitter.com/iabdulkarim472
+github:
+  - url: https://github.com/ArbiLearn
+blockchain:
+  - address: "0x85ba183Fb4c16D435B076DDe459dAfc8eFF864d8"
+    networks:
+      - arbitrum_one
```

Comment:
this is 100% fine, but also note that EOAs can have an `any_evm` tag, to enable indexing on any network (not just `arbitrum_one`)